### ISSUE TITLE
Need maintainers?

### ISSUE THREAD

**Issue by Tmarty** [gcuisinier/jenv#227]
As of writing this it's been [11 months](https://github.com/gcuisinier/jenv/commits/master) since the last change to the master branch. Meanwhile there are many open [pull requests](https://github.com/gcuisinier/jenv/pulls) and [issues](https://github.com/gcuisinier/jenv/issues). Does the project need more maintainers?

**Comment by uded** [comment]
I would risk saying it does and it's kinda frightening...

**Comment by doctorpangloss** [comment]
@Tmarty 

I've merged some of the pull requests and made far more legible documentation here:

https://github.com/hiddenswitch/jenv

**Comment by gcuisinier** [comment]
Indeed, as explained in #228, I don't have a lot of time since .. too long :)

If any other want to work with me on this project too, I propose to move the gcuisinier/jenv to jenv/jenv repository.
And adding interested people as maintainer too.

**Comment by Tmarty** [comment]
Sounds great! I would love to help out in any way that I can. And @doctorpangloss seems interested as well.

**Comment by uded** [comment]
IMHO anything that can help. If one person is not enough maybe two-three main maintainers would make this such a helpful tool moving forward and reacting to external changes...

**Comment by gcuisinier** [comment]
@doctorpangloss What do you think ? Do you prefer to keep your fork  ?
Or are you interested to be a maintainer on this one ?

**Comment by guilgaly** [comment]
Hi, has there been any news on the future of this project since the last comment here?
